lib/antora/versions-sorter-extension.js

```diff
@@ -26,16 +27,28 @@ module.exports.register = function () {
         logComponentsVersions('Sort done', contentCatalog);
     })
 
+    /**
+     * Return true if a version is defined as alpha in its version name or prerelease info.
+     * @param version The version object like one from component.versions array
+     */
+    function isAlpha(version){
+        const explicitPrerelease = version.prerelease
+        const res = (typeof explicitPrerelease === "string" && explicitPrerelease.includes('alpha'))
+                || (version.version.includes('alpha'))
```

Comment:
**thought:** I addition, we could support "Alpha" version (1st letter upper-cased)
We could do the check on lowercase versions